frontend/packages/console-app/src/components/cloud-shell/Terminal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { Terminal as XTerminal } from 'xterm';
+import { fit } from 'xterm/lib/addons/fit/fit';
+
+const terminalOptions = {
+  fontFamily: 'monospace',
+  fontSize: 16,
+  cursorBlink: false,
+  cols: 80,
+  rows: 25,
+  padding: 4,
```

Comment:
Padding is not an option on `ITerminalOptions`. This must be a feature only available in later releases.
We need to find a way to add it by some other means or for now just remove the padding.